### PR TITLE
Feature/improve sourcery

### PR DESCRIPTION
**Pull request by siemensikkema**
* Fix sourcery submodule
* Adapt to improved sourcery LinuxMain.swift functionality
* Update README

### CODE REVIEW

File: Config/droplet.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "//": "The type of server to use for handling requests.",
+	"//": "The type of server to use for handling requests.",
```
Comment (steffendsommer) [inline Config/droplet.json]: Indentation

File: Tests/LinuxMain.swift [hunk 2]
```diff
@@ -3,10 +3,26 @@
 import XCTest
 @testable import AppTests
 
+// sourcery:inline:auto:LinuxMain
```
Comment (steffendsommer) [inline Tests/LinuxMain.swift:6]: Is this in nodes-vapor/sourcery-templates?

**Comment by siemensikkema** [comment]
@steffendsommer git submodules is breaking travis ...
`Permission denied (publickey)`

**Comment by siemensikkema** [comment]
@steffendsommer I think I fixed the issue by enabling travis on sourcery-templates (configured with it building automatically)

**Review by steffendsommer — APPROVED** [review]

**Pull request merged**